### ISSUE TITLE
Stop Windows Appearing

### ISSUE THREAD

**Issue by tatorth0t** [Unlucky-Life/ankimon#53]
Is there a way to stop the popup windows saying "Pokemon gained xxx Exp. yyy needed to evolve", etc? I like using the 8BitDo controller for anki, but every time one of these windows appears, I have to reclick on the anki window with my mouse. Given I partially use the remote so I don't have to be near the computer, this is very inconvenient. Is there any way to disable the popup windows so that I wouldn't have to keep clicking on the anki window again afterwards?

**Comment by Unlucky-Life** [comment]
yes there is ! Go to the the addon overview => there you can click on configure on "Ankimon by Unlucky-Life"
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/397bb418-24a5-4b4e-8454-0d9cb1aa4679)
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/99caf56c-1e2a-409b-a48d-b60500baf6f8)
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/1b7efc29-7a0f-43fc-a230-3536b0983f87)
=> here you can set pop_up_dialog_message_on_defeat to false
Then the pop up window should dissapear :) !

**Comment by tatorth0t** [comment]
Amazing, thank you so much! 

I still occasionally get a popup whenever I get an item, like a berry. Is there a way to disable that?
